R/getPlot.R

```diff
@@ -0,0 +1,133 @@
+#' Get a plot
+#'
+#' Gets a plot for an FcsFile. If in RStudio or Jupyter and `raw`
+#' is not `TRUE`, displays the image. Otherwise, returns the raw,
+#' PNG-encoded bytes.
```

Comment:
This needs to be updated again after the latest interface change.
```suggestion
#' Gets a plot for an FcsFile. If in RStudio or Jupyter and `display` is `TRUE`
#' (default), displays the image. If `destination` is provided, also saves the
#' image as a PNG file. In all cases, returns the PNG-encoded bytes (invisibly,
#' if `display` is `TRUE`).
```

Reply:
:+1: